src/components/ha-label.ts

```diff
@@ -44,6 +44,7 @@ class HaLabel extends LitElement {
           color: var(--ha-label-text-color);
           --mdc-icon-size: 12px;
           text-wrap: nowrap;
+          font-family: Roboto, sans-serif;
```

Comment:
I'll revert it, especially since I can't find the place it didn't work in... :-) But I will fix ha-input-chip instead. That definitely needs it and it gets Arial if not using their font CSS variable.

Reply:
@bramkragten I reverted it. Waiting for your approval.